osu.Game.Tests/Visual/TestCaseBeatmapScoresContainer.cs

```diff
@@ -47,9 +47,7 @@ public TestCaseBeatmapScoresContainer()
             AddStep("scores pack 1", () => scoresContainer.Scores = scores);
             AddStep("scores pack 2", () => scoresContainer.Scores = anotherScores);
             AddStep("only top score", () => scoresContainer.Scores = new[] { topScore });
-            AddStep("remove scores", scoresContainer.CleanAllScores);
-            AddStep("turn on loading", () => scoresContainer.IsLoading = true);
-            AddStep("turn off loading", () => scoresContainer.IsLoading = false);
```

Comment:
So now there is no way to test loading of scores. This needs a test.

Reply:
There is no loading any more. I removed this because it was broken (go check it before this PR).

Reply:
Well, sure there is no more `IsLoading` property, but the loading animation + retrieval of scores is still there.